packages/react/hover-card/src/HoverCard.stories.tsx

```diff
@@ -0,0 +1,676 @@
+            }}
+          />
+          <HoverCardContent className={chromaticContentClass} side={side} align={align}>
+            <p style={{ textAlign: 'center' }}>
+              {side}
+              <br />
+              {align}
+            </p>
+            <HoverCardArrow className={chromaticArrowClass} width={20} height={10} />
+          </HoverCardContent>
+        </HoverCard>
+      ))
+    )}
+
+    <h1 style={{ marginTop: 100 }}>With slotted trigger</h1>
+    <HoverCard open>
+      <HoverCardTrigger as={Slot}>
+        <button className={triggerClass}>open</button>
+      </HoverCardTrigger>
+      <HoverCardContent className={contentClass} sideOffset={5}>
+        <HoverCardArrow className={arrowClass} width={20} height={10} offset={10} />
+        Some content
+      </HoverCardContent>
+    </HoverCard>
+
+    <h1 style={{ marginTop: 100 }}>State attributes</h1>
+    <h2>Closed</h2>
+    <HoverCard open={false}>
+      <HoverCardTrigger className={triggerAttrClass}>open</HoverCardTrigger>
+      <HoverCardContent className={contentAttrClass} sideOffset={5}>
+        <HoverCardArrow className={arrowAttrClass} width={20} height={10} />
+        Some content
+      </HoverCardContent>
+    </HoverCard>
+
+    <h2>Open</h2>
+    <HoverCard open>
+      <HoverCardTrigger className={triggerAttrClass}>open</HoverCardTrigger>
+      <HoverCardContent className={contentAttrClass} side="right" sideOffset={5}>
+        <HoverCardArrow className={arrowAttrClass} width={20} height={10} />
+        Some content
+      </HoverCardContent>
+    </HoverCard>
+  </div>
+);
+Chromatic.parameters = { chromatic: { disable: false } };
+
+const triggerClass = css({});
+
+function CardContentPlaceholder() {
+  return (
+    <div style={{ maxWidth: 400, display: 'flex', alignItems: 'center' }}>
+      <div style={{ width: 60, height: 60, backgroundColor: 'white', borderRadius: 100 }} />
+      <div style={{ marginLeft: 14 }}>
+        <div style={{ width: 200, backgroundColor: 'white', height: 14, borderRadius: 100 }} />
+        <div
+          style={{
+            width: 150,
+            backgroundColor: 'white',
+            height: 14,
+            borderRadius: 100,
+            marginTop: 10,
+          }}
+        />
+      </div>
+    </div>
+  );
+}
+
+const RECOMMENDED_CSS__HOVERCARD__CONTENT = {
+  transformOrigin: 'var(--radix-hovercard-content-transform-origin)',
+};
+
+const contentClass = css({
+  ...RECOMMENDED_CSS__HOVERCARD__CONTENT,
+  backgroundColor: '$gray300',
+  padding: 20,
+  borderRadius: 5,
+});
+
+const arrowClass = css({
+  fill: '$gray300',
+});
+
+const fadeIn = css.keyframes({
+  from: { transform: 'scale(0.9)', opacity: 0 },
+  to: { transform: 'scale(1)', opacity: 1 },
+});
+
+const fadeOut = css.keyframes({
+  from: { transform: 'scale(1)', opacity: 1 },
+  to: { transform: 'scale(0.9)', opacity: 0 },
+});
+
+const animatedContentClass = css(contentClass, {
+  transformOrigin: 'top center',
```

Comment:
We should remove this, as it is actually overriding the more accurate one from line `598`.

The reason it's a CSS custom property is because it's calculated dynamically from the our popper solution to be the perfect point at the arrow tip. This will update correctly also based on all side, align, and collisions (when it swaps side, etc).

Reply:
The `Popper` prim is just great